oc_cdtapi/BitbucketAPI.py

```
@@ -0,0 +1,211 @@
+        logging.debug('repo_slug: {0}'.format(repo_slug))
+        
+        req = ['projects', project, 'repos', repo_slug]
+        
+        response = self.get(req, headers=self.headers, timeout=self.DEFAULT_TIMEOUT)
+        repo_data = response.json()
+        
+        logging.debug('Retrieved repository: %s', repo_data.get('name', 'unknown'))
+        
+        return repo_data
+    
+    def create_repo(self, project, repo_name, description=None, forkable=True, public=False):
```

Ruff (0.15.12)

[warning] Boolean default positional argument in function definition

(FBT002)